network-api/networkapi/buyersguide/templates/primary_nav.html

```diff
@@ -0,0 +1,56 @@
+<div id="primary-nav-container">
```

Comment:
following the code hierarchy pattern we have for the main Foundation site nav (`network-api/networkapi/templates/pages/primary_nav.html`)